crud.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.orm import Session
+
+from db import models
+import schemas
+
+
+def get_book_list(db: Session, author_id: str | None = None):
```

Comment:
Make sure author_id is of type int if your database schema uses integers for author IDs.